src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -187,6 +190,24 @@ class ObservationsController(
     return ListAssignedPlotsResponsePayload(payloads)
   }
 
+  @ApiResponse(
+      responseCode = "200",
+      content =
+          [
+              Content(
+                  mediaType = "application/gpx+xml",
+                  schema = Schema(type = "string", format = "binary"))])
+  @GetMapping("/{observationId}/plots", produces = ["application/gpx+xml"])
+  @Operation(summary = "Exports monitoring plots assigned to an observation as a GPX file.")
+  fun exportAssignedPlots(@PathVariable observationId: ObservationId): ResponseEntity<ByteArray> {
+    val observations = observationStore.fetchObservationPlotDetails(observationId)
+    val waypoints = observations.flatMap { it.gpxWaypoints() }
+
+    val filename = "observation.gpx"
```

Comment:
Would it make sense to include the observation ID or date or something in the filename to avoid filename collisions if people do this for multiple observations over time?